cl/users/signals.py

```diff
@@ -71,3 +79,35 @@ def complaint_handler(
             email["emailAddress"] for email in complained_recipients
         ]
         handle_complaint(recipient_emails)
+
+
+@receiver(
+    post_save,
+    sender=settings.AUTH_USER_MODEL,
+    dispatch_uid="create_auth_token",
+)
+def create_auth_token(sender, instance=None, created=False, **kwargs):
+    if created:
+        Token.objects.create(user=instance)
+
+
+@receiver(
+    post_save,
+    sender=settings.AUTH_USER_MODEL,
+    dispatch_uid="create_superuser_profile_object",
+)
+def superuser_creation(sender, instance, created, **kwargs):
+    # Create a profile whenever createsuperuser is run
+    if created and instance.is_superuser:
+        UserProfile.objects.create(
+            user=instance,
+            activation_key=sha1_activation_key(instance.username),
+            key_expires=now() + timedelta(days=5),
```

Comment:
Testing this signal, I think it would be good to set `email_confirmed=True` otherwise the superuser won't be able to log in, at least in development since they won't receive the verification email.
```suggestion
            key_expires=now() + timedelta(days=5),
            email_confirmed=True,
```